ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/displayedgraph/ProcessStatus.scala

```diff
@@ -11,42 +11,46 @@ import pl.touk.nussknacker.restmodel.processdetails.ProcessDeploymentAction
 
 //TODO: Do we really  we need ProcessStatus and ProcessState - Do these DTO's do the same things?
 @JsonCodec case class ProcessStatus(status: StateStatus,
-                                    deploymentId: Option[String] = Option.empty,
-                                    allowedActions: List[ProcessActionType] = List.empty,
-                                    icon: Option[URI] = Option.empty,
-                                    tooltip: Option[String] = Option.empty,
-                                    startTime: Option[Long] = Option.empty,
-                                    attributes: Option[Json] = Option.empty,
-                                    errorMessage: Option[String] = Option.empty)
+                                    name: String,
```

Comment:
what is this name? Is it sth different from status.name??

Reply:
It's humanity version of status.. ;) In that way you can extend for example FlinkManager and overide only displayable name.. Mayby  displayableName or displayName will be better namespace for it..? @arkadius WDYT? :)